### DISCUSSION TITLE
Help with Timeout command

### DISCUSSION THREAD

**Discussion by v8ive** [Pycord-Development/pycord#861] [Q&A] [answered]
I understand this is PyCord which is slightly different from Discord.py, however I was under the understanding they worked hand in hand? Was I wrong?
Either way, I used the Timeout command template to create my timeout command but I keep getting this error:

Command raised an exception: AttributeError: 'Member' object has no attribute 'timeout_for'

Previously I was getting an error saying "failed to convert 'int' object from 'minutes' ", however that issue seems to have gone.

Here's the code, any help is appreciated:

![image](https://user-images.githubusercontent.com/62786822/151105552-334b927f-1807-4ec4-bb90-b0b285e061d6.png)

**Comment by BobDotCom** [comment, marked as answer]
Members do have the timeout_for attribute in the newest version. Try checking `discord.__version__`, which should be 2.0.0a. If it's not, try uninstalling and then reinstalling from this github repo.

**Reply by v8ive** [reply]
Thank you! Been busy with the bot sorry for the late response. Had issues with it not grabbing the latest release but got it now :)

**Comment by pintermor9** [comment]
Pass minutes to timeout_for
